### PR TITLE
Display name instead of attribute key as label in cart item data

### PR DESCRIPTION
**Pull request by Dartui**

### CODE REVIEW

File: includes/class-wc-cart.php [hunk 1]
```diff
@@ -594,7 +597,8 @@ public function get_item_data( $cart_item, $flat = false ) {
 				unset( $item_data[ $key ] );
 				continue;
 			}
-			$item_data[ $key ]['key']     = ! empty( $data['key'] ) ? $data['key'] : $data['name'];
+			$item_data[ $key ]['key']     = ! empty( $data['key'] ) ? $data['key'] : $key;
+			$item_data[ $key ]['name']    = ! empty( $data['name'] ) ? $data['name'] : $data['key'];
```
Comment (Dartui) [inline includes/class-wc-cart.php]: I don't know if this is good, but couldn't come up with anything else

**Comment by Dartui** [comment]
Resolves #11870

**Comment by mikejolley** [comment]
I don't really want to introduce another key here - can you explain why you cannot just have the one real value and then filter the display value? As I said earlier, it cannot store both.

**Comment by mikejolley** [comment]
Closing for reasons above. We should not add values which cannot be stored. We should instead allow you to filter the value you want to display based on the value you are storing. Cart and order display should be identical so that once the order is placed, the value does not appear to change.

**Pull request closed without merging**